dtrace-parser/src/dtrace.pest

```diff
@@ -17,10 +17,11 @@ IDENTIFIER = @{ ASCII_ALPHA+ ~ (ASCII_ALPHANUMERIC | "_")* }
 BIT_WIDTH = @{ "8" | "16" | "32" | "64" }
 SIGNED_INT = ${ "int" ~ BIT_WIDTH ~ "_t" }
 UNSIGNED_INT = ${ "uint" ~ BIT_WIDTH ~ "_t" }
+INTEGER = ${ (SIGNED_INT | UNSIGNED_INT) }
```

Comment:
just observing that we can't pass `char` as a parameter type... which is probably fine